src/test/common/test_time.cc

```diff
@@ -56,7 +56,8 @@ static constexpr double bd = bs + ((double)bns / 1000000000.);
 
 template<typename Clock>
 static void system_clock_sanity() {
-  static const typename Clock::time_point brt(seconds(bs) + nanoseconds(bns));
+  static typename Clock::time_point brt(seconds(bs)
```

Comment:
i don't see anything could possibly be wrong here? and for consistency, we might need to leave the "+" at the end of the line instead of starting with it.